client/src/components/SendEmail.js

```diff
@@ -1,42 +1,87 @@
-import React from 'react';
+// import React from 'react';
+
+// const SendEmail = (props) => {
+//     const [email] = React.useState(localStorage.getItem('email'));
+//     const [to] = React.useState(props.trainee.email);
+//     const [subject] = React.useState("Study Buddy");
+//     const [message] = React.useState("Hello, I would like to study with you.");
+
+//     const sendEmail = async (e) => {
+//         e.preventDefault();
+//         const data = {
+//             email,
+//             to,
+//             subject,
+//             message
+//         };
+//         if (email === "" || to === "" || subject === "" || message === "") {
```

Comment:
if we are not using this code, then this shouldn't be left here commented out.

Reply:
unused code is now deleted